doc/src/devdocs/pkgimg.md

```diff
@@ -0,0 +1,47 @@
+# Package Images
+
+Julia package images provide object (native code) caches for Julia packages.
+They are similar to Julia's [system image](@) and support many of the same features.
+In fact the underlying serialization format is the same, and the system image is the base image that the package images are build against.
+
+## High-level overview
+
+Package images are shared libraries that contain both code and data. Like `.ji` cache files they are generated per package. The data section contains both global data (global variables in the package) as well as the necessary metadata about what methods and types are defined by the package. The code section contains native objects that cache the final output of Julia's LLVM based compiler.
+
+The command line option `--pkgimage-native-code=no` can be used to turn-off object caching for this session. Note that this means that cache files have to be regenerated.
```

Comment:
```suggestion
The command line option `--pkgimage-native-code=no` can be used to turn off object caching for this session. Note that this means that cache files have to be regenerated.
```

Reply:
> Note that this means that cache files have to be regenerated.

Does that mean if I turn it off, all existing cache files are discarded? Some clarification would be helpful.

Reply:
We keep up to 10 files for the same source